grammar.pegjs

```diff
@@ -77,7 +77,7 @@ builtinCommandName
  = '[[' / '['
 
 argument "command argument"
- = commandName
+ = concatenation
```

Comment:
Here a concatenation seems fits better. 
A commandName explicit excludes variableAssignment keyword or redirect rules.
Maybe we have to exclude only keyword and redirect rules?